tasks/sep/006-0-sc-guardian-updates/VALIDATION.md

```diff
@@ -0,0 +1,74 @@
+- All key values match the semantic meaning provided, which can be validated using the storage layout links provided.
+
+## State Overrides
+
+The following state overrides should be seen:
+
+### `0xa87675ebb9501C7baE8570a431c108C1577478Fa` (The Security Council Safe)
+
+Links:
+- [Etherscan](https://sepolia.etherscan.io/address/0xE75Cd021F520B160BF6b54D472Fa15e52aFe5aDD)
+
+Enables the simulation by setting the threshold to 1:
+
+- **Key:** `0x0000000000000000000000000000000000000000000000000000000000000004` <br/>
+  **Value:** `0x0000000000000000000000000000000000000000000000000000000000000001`
+
+## State Changes
+
+### `0x54e8baccc67fa3c6b3e9a94baa4d70d1668f0820` (`LivenessGuard`)
+
+Links:
+- [Etherscan](https://sepolia.etherscan.io/address/0x54e8baccc67fa3c6b3e9a94baa4d70d1668f0820) (not verified)
+
+State Changes:
+- **Key:** `0x023c107741c8297b7d31ce48158a7d3e0c1bab34eb099ba05b48e6e0bb5b0324` <br/>
+  **Before:** `0x000000000000000000000000000000000000000000000000000000006620aaa4` <br/>
+  **After:** `0x000000000000000000000000000000000000000000000000000000006621dcfe` <br/>
+  **Meaning:** The 'lastLive' timestamp of the caller has been updated. The key can be verified by:
+    1. Seeing that the [slot of the mapping is 0](https://github.com/ethereum-optimism/optimism/blob/develop/packages/contracts-bedrock/snapshots/storageLayout/LivenessGuard.json#L6).
+    2. obtaining the storage key with `cast`, where the address provided should be the one used to simulate.
+      ```shell
+        cast index address 0x2e2e33fedd27fdecfc851ae98e45a5ecb76904fe 0
+        0x023c107741c8297b7d31ce48158a7d3e0c1bab34eb099ba05b48e6e0bb5b0324
+      ```
+
+- **Key**: `0x56ee16ca3ade18209faccff732edefbb77524a2f2c0c642df2abe4924871e783` <br/>
+  **Before**: `0x000000000000000000000000000000000000000000000000000000006620aaa4` <br/>
+  **After**: `0x0000000000000000000000000000000000000000000000000000000000000000` <br/>
+  **Meaning:** The 'lastLive' timestamp of the removed owner has been deleted. The key can be verified by:
+    1. Seeing that the [slot of the mapping is 0](https://github.com/ethereum-optimism/optimism/blob/develop/packages/contracts-bedrock/snapshots/storageLayout/LivenessGuard.json#L6).
+    2. obtaining the storage key with `cast`, where the address provided should be the one used to simulate.
+      ```shell
+        cast index address 0x78339d822c23D943E4a2d4c3DD5408F66e6D662D 0
+        0x56ee16ca3ade18209faccff732edefbb77524a2f2c0c642df2abe4924871e783
+      ```
+
+### `0xa87675ebb9501c7bae8570a431c108c1577478fa` (`GnosisSafe`)
+
+Links:
+- [Etherscan](https://sepolia.etherscan.io/address/0xa87675ebb9501c7bae8570a431c108c1577478fa)
+
+State Changes:
+
+- **Owner count:** reduced from 10 to 9, because of the removed owner.
+- **Threshold:** increased from 1 to 2 (this change only occurs due to the state override required for the simulation)
+- **Owners mapping:** The deployer `0x7833...` removed from the linked list of owners.
```

Comment:
```suggestion
- **Owners mapping:** The deployer `0x78339d822c23D943E4a2d4c3DD5408F66e6D662D` removed from the linked list of owners.
```

Suggesting this because when I was doing a ctrl + f earlier I couldn't find any references to this address.